npc/re/scripts_athena.conf

```diff
@@ -13,8 +13,8 @@ npc: npc/re/airports/izlude.txt
 npc: npc/re/battleground/bg_common.txt
 
 // --------------------------- Cities ---------------------------
+npc: npc/cities/brasilis.txt
```

Comment:
imo, we should follow the current structure
re files in `npc/re/scripts_athena.conf`
prere files in `npc/prere/scripts_athena.conf`
same content in `npc/scripts_athena.conf`

= brasilis npcs should be duplicated. Then uncommented in `npc/re/scripts_athena.conf`, and commented in `npc/prere/scripts_athena.conf`
or
brasilis npcs should be common to both mode but removed via delnpc in `npc/prere/scripts_athena.conf` by default